### ISSUE TITLE
Role not inside the required context in course list and faq components

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#603]
The pages in this Siteimprove report have elements that have a role that only makes sense in the context of another role: https://my2.siteimprove.com/Accessibility/7333/NextGen/Issue/1?ruleId=42&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2

One such component is [the course list component ](https://my2.siteimprove.com/Inspector/7333/A11Y/Page?pageId=65098641071&impmd=D97548790CE293BDC08F6D9B84D3E607&conf=a+aa+aria+si&issue=cantTell+failed&wcag=twopointone#/sia-r42/failed/NkY4Mjh0bTM=;Y2U0YjYwNjE=)which has elements with the `role="tab"` but their parent elements do not have a contextual role.
